samples/client.c

```diff
@@ -725,6 +725,10 @@ get_dirty_bitmap(int sock, struct vfio_user_dma_map *dma_region)
            (ull_t)range->iova,
            (ull_t)(range->iova + range->size - 1), bitmap[0]);
 
+    if (bitmap[0] != 0) {
```

Comment:
isn't this hard-coding that the the dirty pages are in the first 8?
If so, maybe that's fine, but comment?

Reply:
Yes, I'm not sure about this - I just used `bitmap[0]` because the above `printf` only prints `bitmap[0]`. Is it correct that the dirty pages are in the first 8?

Reply:
I didn't check the sample code, I was hoping you would :)

Reply:
Can you clarify what I would be asserting?